# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1524591819 25200
#      Tue Apr 24 10:43:39 2018 -0700
# Branch stable
# Node ID 47c803b4b8aee6b56e4a4a853eee6630237fa1e2
# Parent  b5b44037069d818f7af67e4db7bb73c41ea0aa82
test: tag possible test failure with known bug #51804

* ov-java.cc: Tag possible test failure under Oracle Java 7 as known bug.

diff --git a/libinterp/octave-value/ov-java.cc b/libinterp/octave-value/ov-java.cc
--- a/libinterp/octave-value/ov-java.cc
+++ b/libinterp/octave-value/ov-java.cc
@@ -3388,17 +3388,17 @@ Return true if @var{x} is a Java object.
 %! assert (n.toString (), "1");
 
 %!testif HAVE_JAVA; usejava ("jvm") <51804>
 %! n = javaObject ("java.lang.Integer", 1.35);
 %! assert (n.compareTo (0), 1);
 %! assert (n.compareTo (1), 0);
 %! assert (n.compareTo (2), -1);
 
-%!testif HAVE_JAVA; usejava ("jvm")
+%!testif HAVE_JAVA; usejava ("jvm") <51804>
 %! n = javaObject ("java.lang.Short", 1.35);
 %! assert (n.compareTo (0), 1);
 %! assert (n.compareTo (1), 0);
 %! assert (n.compareTo (2), -1);
 
 %!testif HAVE_JAVA; usejava ("jvm")
 %! n = javaObject ("java.lang.Byte", int8 (17));
 %! assert (n.compareTo (int8 (20)), -3);
